ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/HttpTimeout.kt

```diff
@@ -292,7 +292,7 @@ public fun convertLongTimeoutToIntWithInfiniteAsZero(timeout: Long): Int = when
 }
 
 /**
- * Converts long timeout in milliseconds to long value. To do that, we need to consider [HttpTimeout.INFINITE_TIMEOUT_MS]
+ * Converts long timeout in milliseconds to long value. To do that, we need to consider [HttpTimeoutConfig.INFINITE_TIMEOUT_MS]
  * as zero and convert timeout value to [Int].
```

Comment:
🙏  Could you also change here `[Int]` to `[Long]`?
```suggestion
 * as zero and convert timeout value to [Long].
```

Reply:
Hi
I inadvertently closed this PR trying to update my branch with new changes. So I raised a new one here -> https://github.com/ktorio/ktor/pull/4774

I hope that it is fine